azure-iot-device/azure/iot/device/common/errors_paho.py

```diff
@@ -0,0 +1,165 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+
+class OperationCancelledError(Exception):
```

Comment:
>OperationCancelledError [](start = 6, length = 23)

there is an `OperationCancelledError`  in the other file. May be we should rename this to` PahoOperationCancelledError`

Reply:
these errors have been removed in favor of the more generic counterparts.


---
In reply to: [302682009](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r302682009) [](ancestors = 302682009)